eirene.yaml

```diff
@@ -0,0 +1,52 @@
+---
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: false
+tools:
+  - name: recetox_aplcms_generate_feature_table
+    owner: recetox
+    tool_panel_section_label: EIRENE
```

Comment:
Do users really know what EIRENE is and will find tools in this category? Maybe a more scientific label would be more useful?

Reply:
I think the whole Exposome debate is too heated to classify tools as "Exposomics" only, so I think having an EIRENE section is cleaner at the moment. We will add Exposomics to the Ontology topics and maybe also to the toolshed so that we can classify the tools there? this should make sure they are findable.